packages/navie/src/prompt.ts

```diff
@@ -150,5 +158,5 @@ export function buildPromptDescriptor(promptType: PromptType): string {
 
 export function buildPromptValue(promptType: PromptType, value: string): string {
   const prompt = PROMPTS[promptType];
-  return `[${prompt.prefix}] ${value}`;
+  return [`<${prompt.tagName}>`, value, `</${prompt.tagName}>`].join('\n');
```

Comment:
Switching to a syntax here where there is both an opening and closing tag. XML-style tags seem universal for beginning and ending content.